test/acceptance/features/workloadResourceMapping.feature

```diff
@@ -0,0 +1,643 @@
+Feature: Bind services to workloads based on workload resource mapping
+
+    As a user, I would like to be able to use workload resource bindings as defined by the specification.
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        And Service Binding Operator is running
+        And CustomResourceDefinition backends.stable.example.com is available
+        And OLM Operator "not_a_pod" is running
+        And OLM Operator "custom_app" is running
+        And The workload resource mapping is present
```

Comment:
The step should match the step definition - that is `The Workload Resource Mapping is present`

Reply:
Fixed.